libs/kerla_utils/id_table.rs

```diff
@@ -1,21 +1,22 @@
-use crate::bitmap::BitMap;
+use core::mem::size_of;
+use bitvec::prelude::*;
 
-pub struct IdTable<const SZ: usize>(BitMap<SZ>);
+pub struct IdTable<const BIT_LENGTH: usize>(BitArray<[usize; BIT_LENGTH], LocalBits>); // const expr arithmetic unstable
```

Comment:
```suggestion
pub struct IdTable<const BIT_LENGTH: usize>(BitArray<[usize; BIT_LENGTH], LocalBits>);
```

I'm not sure what you want to point out here. Do you mean we need a feature gate to use this expression?

Reply:
@nuta I wanted to calculate compile time the size of the BitArray from the number of required bits, but the feature isn't good enough to use even with the nightly at the moment. I can remove that comment...

Reply:
Makes sense. Rust's const fn is still developing so let's leave this comment. Thanks!